<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectServiceDialog</class>
 <widget class="QDialog" name="SelectServiceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>578</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
  height: 500px;
  width: 400px;
  background-color: #b59683;
}

#frame {
  border: 3px solid #ebebd8;
  border-radius: 15%;
  background-color: #ebebd8;
}

QLabel {
	color: #b59683;
	font-weight: bold;
}

QLabel[objectName*=&quot;title&quot;] {
  padding: 5px;
  
  font-size: 42px;
  font-weight: bold;

  color: #eeeee4;
  border-radius: 15%;
  background-color: #b59683;
  
  margin-bottom: 10px;
}

QLabel[objectName*=&quot;error&quot;] {
	color: #284256;

	font-style: italic;
	text-align: center right;

	margin-bottom: 10px;
}

QLabel[objectName*=&quot;image&quot;] {
  	min-height: 150px;
  	min-width: 150px;
  
  	border: solid;
	border-width: 3px;
	border-radius:15%;
	border-color: #b59683;

	background-color: #eeeee4;
}

QPushButton {
	width: 90;
	height: 20;
	padding: 5px;

	font: bold;

	color: #b59683;
	border: solid;
	border-width: 5px;
	border-radius:20px;
	border-color: #b59683;
	background-color: #eeeee4;
}

QPushButton::pressed {
  	color: #eeeee4;
  	background-color: #b59683;
}

QPushButton[objectName*=&quot;save&quot;], QPushButton[objectName*=&quot;upload&quot;]
{
  width: 90;
  height: 20;
  padding: 5px;

  font: bold;

  color: #eeeee4;
  border: solid;
  border-width: 5px;
  border-radius:20px;
  border-color: #284256;
  background-color: #284256;
}

QPushButton[objectName*=&quot;save&quot;]::pressed, QPushButton[objectName*=&quot;upload&quot;]::pressed {
  color: #284256;
  background-color: #eeeee4;
}

QLineEdit {
	width: 200;
	height: 20;
	padding: 5px;

	font: bold;

	color: #284256;
	border: solid;
	border-width: 5px;
	border-radius:20px;
	border-color: #b59683;
	background-color: #eeeee4;
}

QLineEdit::focus {
	border-color: #284256;
}

QSpinBox, QDoubleSpinBox {
    width: 200px; /* Добавляем единицу измерения (px) к значению ширины */
    height: 20px;
    padding: 5px;

    font: bold;

    color: #284256;
    border: 5px solid #b59683; /* Устанавливаем толщину границы */
    border-radius: 20px;
    background-color: #eeeee4;
}

QSpinBox::focus, QDoubleSpinBox::focus {
  border-color: #284256;
}

QDoubleSpinBox::up-button, QDoubleSpinBox::down-button, QSpinBox::up-button, QSpinBox::down-button {
  width: 0;
  height: 0;
}

QPlainTextEdit {
	width: 200;
	height: 100;
	padding: 5px;

	font: bold;

	color: #284256;
	border: solid;
	border-width: 5px;
	border-radius:20px;
	border-color: #b59683;
	background-color: #eeeee4;
}

QPlainTextEdit::focus {
	border-color: #284256;
}

QComboBox {
  width: 90;
	height: 20;
	padding: 5px;

	font: bold;

	color: #b59683;
	border: solid;
	border-width: 5px;
	border-radius:20px;
	border-color: #b59683;
	background-color: #eeeee4;
}

QComboBox:hover {

}

QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 15px;
    border-left: 5px solid #b59683;
}

QComboBox::down-arrow {
    
}

QComboBox QListView {
	padding: 5;
}

QComboBox::selected {
	color: #ebebd8;
	background-color: #284256;
}

  QAbstractItemView {
  border:3px solid #b59683;
  border-radius: 15%;
  background-color: #eeeee4;
}

QAbstractItemView::item {
	color: #b59683;
	border: 1px solid #b59683;
}

QAbstractItemView::item::selected {
	color: #eeeee4;
	background-color: #284256;
	border-color: #284256;
}
  
  QAbstractItemView::selected {
      color: #eeeee4;
      background-color: #284256;
  }

QAbstractItemView QHeaderView {
	
}

QAbstractItemView QHeaderView::selected {
	
}


QAbstractItemView::section {
	padding: 5px;

	color:#eeeee4;
	background-color: #b59683;

	border: none;
	border-right: 1px solid #ebebd8;
}
  
  QGroupBox {
    color: #b59683;

    margin-top: 25px;

    border: solid;
    border-width: 3px;
    border-radius:20px;
    border-color: #b59683;
    background-color: #eeeee4;


  }

  QGroupBox::title {
    font-weight: bold;
  }

#frame_cost {
  padding: 0px;
  border: solid;
	border-width: 3px;
	border-radius:20px;
	border-color: #b59683;
	background-color: #eeeee4;
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_title">
        <property name="text">
         <string>Послуги</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableView" name="tableView">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_error_services">
        <property name="text">
         <string>Помилка! Послугу не обрано.</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Кількість:</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_error_number">
        <property name="text">
         <string>Помилка! Некоректна кількість.</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>89</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="pushButton_cancel">
          <property name="text">
           <string>Скасувати</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_save">
          <property name="text">
           <string>Додати</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
